x-pack/solutions/security/packages/data-stream-adapter/src/data_stream_adapter.ts

```diff
@@ -5,7 +5,8 @@
  * 2.0.
  */
 
-import { IndexAdapter, SetIndexTemplateParams, type InstallParams } from '@kbn/index-adapter';
+import type { SetIndexTemplateParams } from '@kbn/index-adapter';
```

Comment:
why this separation?

Reply:
Dunno, ask kibanamachine!

Reply:
Just kidding, probably cause I fixed a path in the eslint rules.

Reply:
So weird, we have another type import just below this, from the same file